modules/contact-form/grunion-contact-form.php

```diff
@@ -2026,9 +2031,22 @@ static function parse( $attributes, $content ) {
 			 *
 			 * @param string $class Additional CSS classes for button attribute.
 			 */
-			$submit_button_class = apply_filters( 'jetpack_contact_form_submit_button_class', 'pushbutton-wide' );
+			$submit_button_class = apply_filters( 'jetpack_contact_form_submit_button_class', 'pushbutton-wide' . $gutenberg_submit_button_classes );
+
+			$submit_button_styles = '';
+			if ( isset( $attributes['customBackgroundButtonColor'] ) && $attributes['customBackgroundButtonColor'] ) {
+				$submit_button_styles .= 'background-color: ' . $attributes['customBackgroundButtonColor'] . '; ';
+			}
+			if ( isset( $attributes['customTextButtonColor'] ) && $attributes['customTextButtonColor'] ) {
+				$submit_button_styles .= 'color: ' . $attributes['customTextButtonColor'] . ';';
+			}
+			if ( isset( $attributes['submitButtonText'] ) && $attributes['submitButtonText'] ) {
+				$submit_button_text = $attributes['submitButtonText'];
+			} else {
+				$submit_button_text = $form->get_attribute( 'submit_button_text' );
+			}
 
-			$r .= "\t\t<input type='submit' value='" . esc_attr( $form->get_attribute( 'submit_button_text' ) ) . "' class='" . $submit_button_class . "'/>\n";
+			$r .= "\t\t<input type='submit' value='" . esc_attr( $submit_button_text ) . "' class='" . esc_attr( $submit_button_class ) . "' style='" . esc_attr( $submit_button_styles ) . "'/>\n";
```

Comment:
What happens here if `$submit_button_styles` is an empty string? Should we try to only show `style` when we actually have styles to display, kind of how you did in https://github.com/Automattic/jetpack/pull/11124/files#diff-99eaf41dd1a227df1293c796a7ee6f8bR706 ?